modules/workflow-workflowoperation/src/main/java/org/opencastproject/workflow/handler/workflow/AnalyzeTracksWorkflowOperationHandler.java

```diff
@@ -103,6 +103,9 @@ public WorkflowOperationResult start(WorkflowInstance workflowInstance, JobConte
       properties.put(varName + "_video", Boolean.toString(track.hasVideo()));
       properties.put(varName + "_audio", Boolean.toString(track.hasAudio()));
       properties.put(varName + "_subtitle", Boolean.toString(track.hasSubtitle()));
+      if (track.getDuration() != null) {
+        properties.put(varName + "_duration", Float.toString(track.getDuration().floatValue() / 1000));
```

Comment:
This needs to be documented.

Reply:
Yes, I added the respective documentation to the analyze tracks WOH docs.